angrmanagement/ui/widgets/qfunction_table.py

```diff
@@ -8,6 +8,8 @@
 from PySide2.QtGui import QBrush, QColor
 from PySide2.QtCore import Qt, QSize, QAbstractTableModel, SIGNAL, QEvent
 
+from angrmanagement.data.instance import ObjectContainer
```

Comment:
I prefer a relative import (`from ....data.instance import ObjectContainer`). Why is absolute import better in this case?

Reply:
Because PyCharm does it automatically and then I forget to change it...